cartography/intel/digitalocean/compute.py

```diff
@@ -15,7 +15,7 @@ def sync(
         neo4j_session: neo4j.Session,
         manager: Manager,
         projects_resources: dict,
-        digitalocean_update_tag: str,
+        digitalocean_update_tag: int,
```

Comment:
The update tag really should be either string or int.
https://lyft.github.io/cartography/dev/writing-intel-modules.html#handling-cartography-s-update-tag